src/main/webapp/jsp/instructorCourseEnroll.jsp

```diff
@@ -29,12 +35,33 @@
       <form id="student-data-spreadsheet-form" action="${data.instructorCourseEnrollSaveLink}" method="post"
             class="form-horizontal" role="form">
         <input type="hidden" name="${SESSION_TOKEN}" value="${data.sessionToken}">
+        <input type="hidden" name="${COURSE_ID}" value="${data.courseId}">
+        <input type="hidden" name="${USER_ID}" value="${data.account.googleId}">
         <div class="col-md-12">
           <div class="form-group">
-            <t:statusMessage statusMessagesToUser="${data.statusMessagesToUser}"/>
-            <div id="student-data-spreadsheet">
-              <div class="panel-body">
-                <div id="spreadsheet"></div>
+            <div class="student-existing-data-spreadsheet" id="data-spreadsheet">
+              <div class="panel panel-default">
+                <div class="panel-heading ajax_submit">
```

Comment:
what is this `ajax_submit` class for?

Reply:
It is used to coordinate the collapsing of panels. Usage in `instructorCourseEnrollHelper.js`'s `hideExistingStudentsPanel` and `toggleExistingStudentsPanel` and also a selector to capture a click event in `instructorCourseEnrollPage.js`

Reply:
Can you explain a bit more? I only see the class being added and then removed. What's the point of that?

Reply:
In this current function:

https://github.com/tanhengyeow/teammates/blob/6b8c6daf7159df80a3711588f77bdbaa63ebfddf/src/main/webapp/dev/js/main/instructorCourseEnrollPage.js#L134-L149

It checks if the `ajax_submit` class is present to decide whether to collapse or hide the panel. Adding the class and removing it serves this function.

Reply:
I see, you are manually adding the classes `ajax_submit` and `checked` to keep track of some states. I also see that you are using Bootstrap's `collapse`. Can you just use the classes added by `collapse` to detect instead?